sql/item_basic.sql

```diff
@@ -6629,6 +6661,142 @@ INSERT INTO `item_basic` VALUES (9225,0,'abdhaljs_needle','abdhaljs_needle',99,2
 INSERT INTO `item_basic` VALUES (9226,0,'mog_kupon_a-emi','kupon_a-emi',12,28736,99,0,0);
 INSERT INTO `item_basic` VALUES (9227,0,'mog_kupon_a-syw','kupon_a-syw',12,28736,99,0,0);
 INSERT INTO `item_basic` VALUES (9228,0,'mog_kupon_a-asyw','kupon_a-asyw',12,28736,99,0,0);
+INSERT INTO `item_basic` VALUES (9308,0,'craftsmans_crystal:_woodworking','c._crystal:_wood.',12,24640,99,1,0);
```

Comment:
Some have slipped through the cracks over the years but we normally strip symbols like `:` `'` `,` and `.` (most symbols can't be in the file name anyway when we make scripts for the item)